proto/permissions.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "momento.shared.permissions";
+
+package permissions;
```

Comment:
if we can avoid using a package here, that might be sufficient to keep the web sdk working.

Reply:
I agree that we need to test the web SDK but I don't know that I want to introduce a new inconsistency in our use (or lack thereof) of packages until we know there's a problem with the web SDK. I'd rather be consistent with our current conventions and see if we can solve it for Web SDK.

Reply:
Sure. The hack will almost definitely break though, because even if we remove `package permissions` temporarily, the types in it are still being referred to as `permissions.*` in the other protos.